python/tvm/topi/cuda/batch_matmul.py

```diff
@@ -367,7 +367,7 @@ def _schedule_batch_matmul_int8(cfg, s, output):
     # dp4a tensorize
 
     target = tvm.target.Target.current(allow_none=False)
-    do_tensorize = True
+    do_tensorize = target.supports_integer_dot_product
```

Comment:
I think we can always initialize `do_tensorize` as
```
do_tensorize = "+dotprod" in target.mattr or target.supports_integer_dot_product
```
and remove `if is_target(["vulkan", "rocm"]):` below.

Reply:
Done, not sure, why the tests are failing.